composer.json

```diff
@@ -57,19 +57,17 @@
         "doctrine/dbal": "^3.2",
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/mongodb-odm": "^2.3",
-        "doctrine/orm": "^2.14.0 || ^3.0",
-        "friendsofphp/php-cs-fixer": "^3.14.0",
-        "nesbot/carbon": "^2.71 || ^3.0",
-        "phpstan/phpstan": "^1.11",
-        "phpstan/phpstan-doctrine": "^1.4",
-        "phpstan/phpstan-phpunit": "^1.4",
-        "phpunit/phpunit": "^9.6",
-        "rector/rector": "^1.1",
-        "symfony/console": "^5.4 || ^6.0 || ^7.0",
-        "symfony/doctrine-bridge": "^5.4 || ^6.0 || ^7.0",
-        "symfony/phpunit-bridge": "^6.0 || ^7.0",
-        "symfony/uid": "^5.4 || ^6.0 || ^7.0",
-        "symfony/yaml": "^5.4 || ^6.0 || ^7.0"
+        "doctrine/orm": "^2.14.0",
```

Comment:
Please, check the whole file. Many of these changes seem like a regression.

Reply:
Thanks for your patience, this is my first time contributing to a large project. I've gone through the whole file and updated all packages to match the current versions.

Reply:
Thanks for taking the time to iterate and fix, JDruery!